docs/GettingStarted.md

```diff
@@ -153,6 +175,20 @@ Before downloading tracing on your application, install the Datadog Agent. The R
 
 ### Quickstart for Ruby applications
 
+#### Ruby Auto Instrument all Integrations
+
+1. Install the gem with `gem install ddtrace`
+2. Add `require 'ddtrace/auto_instrument'` to your application startup sequence. This must be done _after_ requiring any libraries or frameworks that should be instrumented. 
+
+    ```ruby
+    require 'sinatra'
```

Comment:
If I understand correctly, sinatra is just an example here. We should add a comment about it here, so that folks know they don't need sinatra specifically.

Reply:
Yup, makes sense, i've added a note here explaining this, i think it's clearer now.